### ISSUE TITLE
[BUG]: Cannot download newer docker image after release of 1.29.1

### ISSUE THREAD

**Issue by ssv-w32** [microsoft/playwright#19658]
After release of 1.29.1 started to get message to update docker image:
```
E       playwright._impl._api_types.Error: Executable doesn't exist at /ms-playwright/chromium-1033/chrome-linux/chrome
E       ╔═══════════════════════════════════════════════════════════════╗
E       ║ Looks like Playwright was just updated to 1.28.0.             ║
E       ║ Please update docker image as well.                           ║
E       ║ -  current: mcr.microsoft.com/playwright/python:v1.29.1-focal ║
E       ║ - required: mcr.microsoft.com/playwright/python:v1.28.0-focal ║
E       ║                                                               ║
E       ║ <3 Playwright Team                                            ║
E       ╚═══════════════════════════════════════════════════════════════╝
```

but looks like there is no image available:
```
docker pull mcr.microsoft.com/playwright/python:v1.29.1-focal                                          
Error response from daemon: manifest for mcr.microsoft.com/playwright/python:v1.29.1-focal not found: manifest unknown: manifest tagged by "v1.29.1-focal" is not found
```

**Comment by ssv-w32** [comment]
yep, sure

Playwright
```
python -c "from playwright._repo_version import version;print(version)" 
1.28.0

docker
```
root@40a9cc309e57:/frontend-automation# cat /ms-playwright/.docker-info
{
  "driverVersion": "1.28.0",
  "dockerImageName": "mcr.microsoft.com/playwright/python:v1.28.0-focal"
}
```

```

**Comment by ssv-w32** [comment]
I cannot even get 1.29.1 from Pypi called from local machine:

```
pip install playwright==1.29.1   
                                      
ERROR: Could not find a version that satisfies the requirement playwright==1.29.1 (from versions: 1.9.0, 1.9.1, 1.9.2, 1.10.0, 1.11.0, 1.11.1, 1.11.2, 1.12.0, 1.12.1, 1.13.0, 1.13.1, 1.14.0, 1.14.1, 1.15.0, 1.15.1, 1.15.2, 1.15.3, 1.16.0, 1.16.1, 1.17.0, 1.17.1, 1.17.2, 1.18.0, 1.18.1, 1.18.2, 1.19.0, 1.19.1, 1.20.0, 1.20.1, 1.21.0, 1.22.0, 1.23.0, 1.23.1, 1.24.0, 1.24.1, 1.25.0, 1.25.1, 1.25.2, 1.26.0, 1.26.1, 1.27.0, 1.27.1, 1.28.0, 1.29.0)

```

**Comment by dimkin-eu** [comment]
because python is not in sync with js
there is `node` image mcr.microsoft.com/playwright:v1.29.1-focal
but that doesn't mean there will be `py` image mcr.microsoft.com/playwright/python:v1.29.1-focal and now only FROM mcr.microsoft.com/playwright/python:v1.29.0-focal available

tldr - you need 1.29.0
